src/Components/RecsListTable/RecsListTable.spec.ct.js

```diff
@@ -420,6 +420,21 @@ describe('successful non-empty recommendations list table', () => {
     cy.get(TOOLBAR_FILTER).find('.pf-c-form-control').should('be.empty');
   });
 
+  it('rule content is rendered', () => {
+    // expand all rules
+    cy.get('.pf-c-toolbar__expand-all-icon > svg').click();
+    cy.get(RECS_LIST_TABLE)
+      .find('.pf-c-table__expandable-row.pf-m-expanded')
+      .each((el) => {
+        // contains description
+        cy.wrap(el).contains(
+          'Sed ut perspiciatis unde omnis iste natus error.'
```

Comment:
where is the content defined? Is it coming from one of the already existing fixtures?

Reply:
@ikerreyes, yep, from here https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/cypress/fixtures/RecsListTable/data.json

Reply:
great. Then I think it would be better to check the string from the data itself rather than hardcoding it in the tests, but we can address that later.

Reply:
the test is rather a showcase :) we'll improve it later when focusing more on the testing perspective